test/integration/smart_answer_flows/overseas_passports_test.rb

```diff
@@ -1311,4 +1311,16 @@ class OverseasPassportsTest < ActiveSupport::TestCase
       end
     end
   end
+
+  context "St Martin" do
+    should "suggest to apply online" do
+      worldwide_api_has_no_organisations_for_location('st-martin')
+      add_response 'st-martin'
+      add_response 'renewing_new'
+      add_response 'adult'
+
+      assert_current_node :ips_application_result_online
+      assert_phrase_list :how_long_it_takes, [:how_long_6_weeks, :how_long_additional_info_renewing_new, :how_long_additional_time_online]
+    end
+  end
 end
```

Comment:
It wasn't tested before, because it was not an exceptional case perhaps. I'll add a test and make sure it's clear behaviour has to be the same for both sides of the island :)